portal/src/ScreenLayoutScrollView.module.css

```diff
@@ -3,4 +3,7 @@
   min-width: 0;
   min-height: 0;
   overflow: auto;
+
+  /* Let body to scroll in mobile */
+  @apply mobile:overflow-visible mobile:flex-grow-only mobile:min-h-min;
```

Comment:
Instead of introducing `flex-grow-only`, we can rewrite this block into

```
flex-grow: 1;
flex-shrink: 0;
flex-basis: 0;

@apply mobile:grow mobile:shrink-0 mobile:basis-auto
```

Reply:
Yes, but seems `flex: 1 0 auto` is something we use in a group frequently so I declared it in the tailwind config.

Reply:
Fair enough. What about renaming it to `flex-1-0-auto`?

Reply:
@louischan-oursky Added a commit to rename it to `flex-1-0-auto`